lib/helpers.js

```diff
@@ -216,7 +221,7 @@ function buildStartCmd (startAppOptions, apiLevel) {
   return cmd;
 }
 
-async function getSdkToolsVersion () {
+const getSdkToolsVersion = _.memoize(async function () {
```

Comment:
could you please provide a code example for adding a name to the function that gets memoized?

Reply:
In our code? We don't do it generally, but we should, since it makes reading stack traces easier. Any anonymous function can be made non anonymous and then the name will show up in the trace. Any little help in making babel traces easier to read is a good thing in my book.

Reply:
fixed